core/src/main/resources/application.conf

```diff
@@ -23,6 +23,12 @@ configManager {
 
 ktor {
   deployment {
+    publicScheme = "http"
+    publicScheme = ${?PUBLIC_SCHEME}
+    publicFqdn = "localhost"
+    publicFqdn = ${?PUBLIC_FQDN}
+    publicPort = 8080
+    publicPort = ${?PUBLIC_PORT}
```

Comment:
You need to add those new properties to the config files used for testing at `core/src/test/resources` as well (without the environment variable lines as they are not used anyway).

Reply:
Thanks! I hadn't gotten around to look why the tests failed but now that I did I also wouldn't have found the issue in the output. :)